### PR TITLE
Mono documentation

### PR DESCRIPTION
**Pull request by arschles**
TODO:
- [x] test upload and queue steps
- [ ] ~~add a config file, just as in the java examples~~

cc/ @treeder

### CODE REVIEW

File: mono/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+dependencies are passed into javac.
+
+```sh
+docker run --rm -v "$(pwd)":/worker -w /worker iron/mono:4.0.1 sh -c 'mcs -r:System.Web.Extensions.dll worker101.cs'
+```
+
+Now run it to test it out:
+
+```sh
+docker run --rm -v "$(pwd)":/worker -w /worker iron/mono:4.0.1 sh -c 'mono worker101.exe -payload hello.payload.json -id 123'
```
Comment (treeder) [inline mono/README.md]: Change to env vars for payload.

File: mono/hello.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+using System;
+
+public class HelloWorld
+{
+    static public void Main ()
+    {
+        Console.WriteLine ("Hello Mono World");
```
Comment (treeder) [inline mono/hello.cs]: Add code to use our .NET libs to read payload.

Comment (arschles) [inline mono/hello.cs]: @treeder where are our .Net libs? don't see anything on GH and this code taken from http://dev.iron.io/worker/languages/dotnet/#payload_example

**Comment by arschles** [comment]
@treeder this now uses an env var to get the payload file as requested. none of the client libraries listed in the dev center (http://grcodemonkey.github.io/iron_sharp/, https://github.com/odeits/IronTools and https://github.com/iron-io/iron_dotnet) seem to provide payload lookup so unable right now to follow [your request](https://github.com/iron-io/dockerworker/pull/21#discussion_r40608029). still combing through the code to make sure but it's a line of code to get the value of an env var in C# anyway.

please let me know if there are any other blockers to merging.

**Comment by treeder** [comment]
Should update one of the dotnet libs (whichever is the best one) to read the env vars since that's our primary way of doing things now (regardless of language).

**Comment by arschles** [comment]
@treeder it's on my todo list, but rather merge this now then update

**Comment by treeder** [comment]
Ok, that doesn't look too bad like this anyways, couple lines, I like it.

**Pull request merged**